src/main/java/com/adobe/jenkins/github_pr_comment_build/TriggerPRCommentBranchProperty.java

```diff
@@ -16,14 +16,16 @@ public class TriggerPRCommentBranchProperty extends BranchProperty {
      * The comment body to trigger a new build on.
      */
     private final String commentBody;
+    private final boolean allowUntrusted;
 
     /**
      * Constructor.
      * @param commentBody the comment body to trigger a new build on
      */
     @DataBoundConstructor
-    public TriggerPRCommentBranchProperty(String commentBody) {
+    public TriggerPRCommentBranchProperty(String commentBody, boolean allowUntrusted) {
```

Comment:
No!

Please use databound settter.

This parameter is not mandatory and your breaking constructor compatibility. 

```java
public void SetAllowedUntrusted(boolean allowUntrusted) {
    this.allowUntrusted = allowUntrusted;
}
```

Reply:
Good catch, didn't think about this. Though it should be `setAllowedUntrusted` with a `@DataBoundSetter` annotation, correct?